### PR TITLE
RFC: identity

### PR DESCRIPTION
**Pull request by b5**

### CODE REVIEW

File: text/0005-identity.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+# Summary
+[summary]: #summary
+
+Define the baseline _self-soverign identity system_ for identifying & describing
+peers on qri.
+
+# Motivation
+[motivation]: #motivation
+
+qri is designed to be a system who's trust model is built on trust between
```
Comment (rgardaphe) [inline text/0005-identity.md]: should read: "be a system whose trust model"

File: text/0005-identity.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+qri profiles connect descriptive information about a person or group of people
+that control a keypair
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Profiles are the basis of qri's _self-soverign identity system_, as such it's
+worth defining a few key terms related to self-soverign identity. Much of this
+is drawn from a blog post: [A Gentle Introduction to Self-Soverign Identity](https://bitsonblocks.net/2017/05/17/a-gentle-introduction-to-self-sovereign-identity/)
```
Comment (dustmop) [inline text/0005-identity.md]: URL has changed: https://bitsonblocks.net/2017/05/17/gentle-introduction-self-sovereign-identity/

File: text/0005-identity.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+// Profile is a definition of a profile on qri
+type Profile struct {
+  ID       string `json:"id"`
+  PrivKey  string `json:"privkey,omitempty"`
+  Peername string `json:"peername"`
+  // Created timestamp
+  Created time.Time `json:"created"`
+  // Updated timestamp
+  Updated time.Time `json:"updated"`
+  // specifies weather this is a user or an organization
```
Comment (dustmop) [inline text/0005-identity.md]: whether

What are the specific string values in each case? "user" and "organization"?

File: text/0005-identity.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+  Type string `json:"type"`
+  // user's email address
+  Email string `json:"email"`
+  // user name field. could be first[space]last, but not strictly enforced
+  Name string `json:"name"`
+  // user-filled description of self
+  Description string `json:"description"`
+  // url this user wants the world to click
+  HomeURL string `json:"homeurl"`
+  // color this user likes to use as their theme color
```
Comment (dustmop) [inline text/0005-identity.md]: Is this html-like "#rrggbb"? Are certain web color names allowed?

File: text/0005-identity.md (hunk 3 above)
Comment (ramfox) [reply text/0005-identity.md]: actually `peer` and `organization`

A bunch of this is described in the config.profile validator: https://github.com/qri-io/qri/blob/56a647be2f3dc01a844b144bd181470eeaa95b17/config/profile.go#L85-L216

File: text/0005-identity.md (hunk 4 above)
Comment (b5) [reply text/0005-identity.md]: I think we should go with #rrggbb or empty string (which should represent the default)

**Comment by b5** [comment]
Closing. We've evolved past this RFC

**Pull request closed without merging**